python/grass/script/imagery.py

```diff
@@ -89,22 +85,37 @@ def group_to_dict(
             .strip()
             .split()
         )
-    except CalledModuleError:
-        return group_dict
+    except CalledModuleError as cme:
+        raise cme
```

Comment:
The trend is to avoid names which are not words, i.e., no `strchr` :-)

```suggestion
    except CalledModuleError as error:
        raise error
```

Reply:
Addressed in [85d36a0](https://github.com/OSGeo/grass/pull/3756/commits/85d36a0c83efe0eea2d386b8dd68c970fd326f45)